docs/guide/localization/static.md

```diff
@@ -261,19 +335,19 @@ Here is an example module `index.js` file that sets the `browser: true` flag for
 
 <AposCodeBlock>
 
-  ```javascript
-  module.exports = {
-    // Note this is at top level, not under options
-    i18n: {
-      ourTeamUI: {
-        browser: true
-      }
+```javascript
+module.exports = {
+  // Note this is at top level, not under options
+  i18n: {
+    ourTeamUI: {
```

Comment:
Should be `venueStrings` now to match the JSON files example, yeah?

Reply:
Yeah - would be best for continuity.